src/Core/EmailService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace VPlugins\MalwareScanner\Core;
+
+class EmailService {
+
+    /**
+     * Send scan results via email.
+     *
+     * @param array $report The scan report.
+     * @return bool True if the email was sent successfully, false otherwise.
+     */
+    public static function send_scan_email($report) {
```

Comment:
- Split into three function like `email_header()` and `email_footer()`
- Add $subject param too along with $report

Reply:
Updated